### PR TITLE
Rename MapD -> OmnisciDB

### LINKED ISSUE #3: Replace mapd with omnisci
The following replacements are planned:
`mapd.py` -> `omnisci.py`
`RemoteMapD` -> `RemoteOmnisci` (or `Omnisci`?)

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Fixes #3. Still need to adjust the names on notebooks.

### CODE REVIEW

**Comment by pearu** [comment]
It would be nice to evaluate notebooks as a part of the testing suite.
Created separate issue for that: https://github.com/xnd-project/rbc/issues/13

**Comment by guilhermeleobas** [comment]
Hi @pearu, can you review these changes when you have some time to spare?

File: notebooks/rbc-omnisci-ibis.ipynb [hunk 1]
```diff
@@ -2,39 +2,39 @@
...
-   "execution_count": 2,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
-    "mapd = RemoteMapD()  # RemoteMapD(user=, password=, host=, port=, dbname=)\n",
-    "mapd_ibis = ibis.mapd.connect(user=mapd.user, password=mapd.password,\n",
-    "                              host=mapd.host, port=mapd.port, database=mapd.dbname)"
+    "omni = RemoteOmnisci()  # RemoteOmisci(user=, password=, host=, port=, dbname=)\n",
+    "omni_ibis = ibis.omniscidb.connect(user=omni.user, password=omni.password,\n",
```
Comment (pearu) [inline notebooks/rbc-omnisci-ibis.ipynb]: `ibis` does not have `omniscidb`. Restore it as `mapd`.

File: notebooks/rbc-omnisci-ibis.ipynb [hunk 2]
```diff
@@ -2,39 +2,39 @@
...
-   "execution_count": 3,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ibis.mapd bug workaround:\n",
+    "# ibis.omniscidb bug workaround:\n",
     "import sys\n",
-    "sys.modules['ibis.mapd.operations']._sql_type_names['float32'] = 'float'"
+    "sys.modules['ibis.omniscidb.operations']._sql_type_names['float32'] = 'float'"
```
Comment (pearu) [inline notebooks/rbc-omnisci-ibis.ipynb]: `ibis` does not have `omniscidb`. Restore it as `mapd`.

File: notebooks/rbc-omnisci-ibis.ipynb (hunk 1 above)
Comment (guilhermeleobas) [reply notebooks/rbc-omnisci-ibis.ipynb]: Hi @pearu, `mapD` was renamed to `omniscidb` on ibis 6 months ago. See:
https://github.com/ibis-project/ibis/tree/master/ibis/omniscidb
https://github.com/ibis-project/ibis/commit/9df27958f305a901728b540200bd8fa2820d4625

File: notebooks/rbc-omnisci-ibis.ipynb (hunk 2 above)
Comment (guilhermeleobas) [reply notebooks/rbc-omnisci-ibis.ipynb]: See my comment above.

File: notebooks/rbc-omnisci-ibis.ipynb (hunk 1 above)
Comment (pearu) [reply notebooks/rbc-omnisci-ibis.ipynb]: Ok, let's keep ibis.omniscidb. (Todo for myself: update rbc-dev environment).

File: notebooks/rbc-omnisci-ibis.ipynb (hunk 2 above)
Comment (pearu) [reply notebooks/rbc-omnisci-ibis.ipynb]: noop.

File: notebooks/rbc-omnisci-ibis.ipynb (hunk 1 above)
Comment (guilhermeleobas) [reply notebooks/rbc-omnisci-ibis.ipynb]: I had to install ibis from master to get the latest changes: `pip install git+https://github.com/ibis-project/ibis@master`

**Pull request merged**